### PR TITLE
Add NHS3100

### PR DESCRIPTION
**Pull request by jneiva08**
Created NHS3100 and created MCU_NXP_NTAG

Datasheet: https://www.nxp.com/docs/en/data-sheet/NHS3100.pdf

Other PR:
https://github.com/KiCad/kicad-footprints/pull/2077
https://github.com/pointhi/kicad-footprint-generator/pull/494

print-screen:
nhs3100

---

All contributions to the kicad library must follow the [KiCad library convention](http://kicad-pcb.org/libraries/klc/)

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [x] Provide a URL to a datasheet for the symbol(s) you are contributing
- [x] An example screenshot image is very helpful
- [x] Ensure that the associated footprints match the [official footprint library](https://github.com/kicad/kicad-footprints)
  - A new fitting footprint must be submitted if the library does not yet contain one.
- [x] If there are matching footprint PRs, provide link(s) as appropriate
- [x] Check the output of the Travis automated check scripts - fix any errors as required
- [x] Give a reason behind any intentional library convention rule violation.

### CODE REVIEW

File: MCU_NXP_NTAG.lib [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#
+# NHS3100
+#
+DEF NHS3100 U 0 20 Y Y 1 F N
+F0 "U" -550 550 50 H V C CNN
+F1 "NHS3100" 350 550 50 H V C CNN
+F2 "Package_DFN_QFN:HVQFN-24-1EP_4x4mm_P0.5mm_EP2.6x2.6mm" 1300 -550 50 H I C CNN
+F3 "" 0 -100 50 H I C CNN
+$FPLIST
+ HVQFN*1EP*4x4mm*0.5mm*
```
Comment (chschlue) [inline MCU_NXP_NTAG.lib]: HVQFN\*1EP\*4x4mm\*P0.5mm\*

File: MCU_NXP_NTAG.lib [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+X LB 19 700 300 100 L 50 50 1 1 P
+X P1/CLKOUT 2 -700 300 100 R 50 50 1 1 B
+X LA 20 700 400 100 L 50 50 1 1 P
+X P2/SSEL 3 -700 200 100 R 50 50 1 1 B
+X P6/SCLK 4 -700 -300 100 R 50 50 1 1 B
+X P8/MISO 5 700 -300 100 L 50 50 1 1 B
+X P9/MOSI 6 700 -200 100 L 50 50 1 1 B
+X VDD 7 0 600 100 D 50 50 1 1 W
+X GND 8 0 -600 100 U 50 50 1 1 W
+X ~RST 9 700 -400 100 L 50 50 1 1 I
```
Comment (chschlue) [inline MCU_NXP_NTAG.lib]: ~RESET

File: MCU_NXP_NTAG.lib [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+X P11/SWDIO 16 700 0 100 L 50 50 1 1 B
+X LB 19 700 300 100 L 50 50 1 1 P
+X P1/CLKOUT 2 -700 300 100 R 50 50 1 1 B
+X LA 20 700 400 100 L 50 50 1 1 P
+X P2/SSEL 3 -700 200 100 R 50 50 1 1 B
+X P6/SCLK 4 -700 -300 100 R 50 50 1 1 B
+X P8/MISO 5 700 -300 100 L 50 50 1 1 B
+X P9/MOSI 6 700 -200 100 L 50 50 1 1 B
+X VDD 7 0 600 100 D 50 50 1 1 W
+X GND 8 0 -600 100 U 50 50 1 1 W
```
Comment (chschlue) [inline MCU_NXP_NTAG.lib]: VSS

File: MCU_NXP_NTAG.lib [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+X P10/SWCLK 15 700 -100 100 L 50 50 1 1 B
+X P11/SWDIO 16 700 0 100 L 50 50 1 1 B
+X LB 19 700 300 100 L 50 50 1 1 P
+X P1/CLKOUT 2 -700 300 100 R 50 50 1 1 B
+X LA 20 700 400 100 L 50 50 1 1 P
+X P2/SSEL 3 -700 200 100 R 50 50 1 1 B
+X P6/SCLK 4 -700 -300 100 R 50 50 1 1 B
+X P8/MISO 5 700 -300 100 L 50 50 1 1 B
+X P9/MOSI 6 700 -200 100 L 50 50 1 1 B
+X VDD 7 0 600 100 D 50 50 1 1 W
```
Comment (chschlue) [inline MCU_NXP_NTAG.lib]: VDDBAT

**Comment by jneiva08** [comment]
@chschlue  have made the changes requested.

Screenshot:
nhs

**Comment by jneiva08** [comment]
for reserved pins needs be connected to VSS, need have the same name VSS?  (S4.3)

**Comment by chschlue** [comment]
Yes

**Comment by chschlue** [comment]
Two more things:
- [ ] change `~RST` to `~RESET`
- [ ] add `, VQFN-24` to the description

**Comment by chschlue** [comment]
Thanks.

**Comment by jneiva08** [comment]
Thanks for your time

**Pull request merged**